### PR TITLE
Product cards

### PR DESCRIPTION
**Pull request by ArletteR**
[UI] Card oferta ilimitada #5
[UI] Card producto vertical #6

image

### CODE REVIEW

File: src/app/shared/card-sale/card-sale.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (AnaCastellanosBambu) [inline src/app/shared/card-sale/card-sale.component.spec.ts]: Delete spec.ts

File: src/app/shared/card-sale/card-sale.component.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+.le-sale-card-main {
+    max-width: 344px;
+    color: $color-font;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.16);
+    display: flex;
+    flex-direction: row;
+    .le-sale-card-left {
+        padding-top: 24px;
+        display: flex;
```
Comment (AnaCastellanosBambu) [inline src/app/shared/card-sale/card-sale.component.scss]: Use helper class 'le-row'

File: src/app/shared/card-sale/card-sale.component.scss [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        flex-direction: column;
+        justify-content: space-between;
```
Comment (AnaCastellanosBambu) [inline src/app/shared/card-sale/card-sale.component.scss]: use helper class 'le-justify-sb'

File: src/app/shared/card-sale/card-sale.component.scss [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            margin-left: -8px;
+            width: 101%;
+            mat-icon {
+                margin-right: 8px;
+            }
+        }
+        .le-sale-card-country {
+            padding-left: 8px;
+            display: none;
+            align-items: center;
```
Comment (AnaCastellanosBambu) [inline src/app/shared/card-sale/card-sale.component.scss]: use helper class 'le-align-center'

File: src/app/shared/card-sale/card-sale.component.scss [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        .le-sale-product-image {
+            background-repeat: no-repeat;
+            background-size: contain;
+            background-position: center;
+            height: 136px;
+            width: 136px;
+        }
+    }
+    .le-sale-card-right {
+        text-align: center;
```
Comment (AnaCastellanosBambu) [inline src/app/shared/card-sale/card-sale.component.scss]: use helper class 'le-text-center'

File: src/app/shared/general-card/general-card.component.scss [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+            left: -17px;
+            top: 0;
+        }
+        button {
+            display: none;
+        }
+    }
+    .le-general-card-country {
+        display: none;
+        align-items: center;
```
Comment (AnaCastellanosBambu) [inline src/app/shared/general-card/general-card.component.scss]: Use helper class. Review all.

File: src/app/shared/general-card/general-card.component.spec.ts [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (AnaCastellanosBambu) [inline src/app/shared/general-card/general-card.component.spec.ts]: Delete spec.ts

File: src/app/shared/card-sale/card-sale.component.spec.ts (hunk 1 above)
Comment (ArletteR) [reply src/app/shared/card-sale/card-sale.component.spec.ts]: done

File: src/app/shared/card-sale/card-sale.component.scss (hunk 2 above)
Comment (ArletteR) [reply src/app/shared/card-sale/card-sale.component.scss]: done

File: src/app/shared/card-sale/card-sale.component.scss (hunk 3 above)
Comment (ArletteR) [reply src/app/shared/card-sale/card-sale.component.scss]: done

File: src/app/shared/card-sale/card-sale.component.scss (hunk 4 above)
Comment (ArletteR) [reply src/app/shared/card-sale/card-sale.component.scss]: done

File: src/app/shared/card-sale/card-sale.component.scss (hunk 5 above)
Comment (ArletteR) [reply src/app/shared/card-sale/card-sale.component.scss]: done

File: src/app/shared/general-card/general-card.component.scss (hunk 6 above)
Comment (ArletteR) [reply src/app/shared/general-card/general-card.component.scss]: done

File: src/app/shared/general-card/general-card.component.spec.ts (hunk 7 above)
Comment (ArletteR) [reply src/app/shared/general-card/general-card.component.spec.ts]: done

**Comment by AnaCastellanosBambu** [comment]
💯

**Pull request merged**